components/drops/TimeTag.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div
+    class="tag-container is-flex border py-1 px-2 is-justify-content-space-between is-align-items-center">
+    <div class="icon-box">
+      <img v-if="isMintingLive" :src="unlockablePulse" />
+      <NeoIcon
+        v-else
+        icon="calendar-day"
+        pack="far"
+        variant="k-grey"
+        class="mr-2" />
+    </div>
+
+    <span class="pr-1">{{ displayText }}</span>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import unlockablePulse from '@/assets/unlockable-pulse.svg'
+import { NeoIcon } from '@kodadot1/brick'
+const { $i18n } = useNuxtApp()
+const props = defineProps<{
+  dropStartTime: Date
+}>()
+
+const isMintingLive = computed(() => {
+  const now = new Date()
+  return props.dropStartTime <= now
+})
+
+const displayText = computed(() => {
+  if (isMintingLive.value) {
+    return $i18n.t('drops.mintingLive')
+  } else {
+    const options = {
+      day: '2-digit',
+      month: '2-digit',
+      hour: '2-digit',
+      minute: '2-digit',
+      hour12: false,
+    } as any
+    return props.dropStartTime.toLocaleString($i18n.locale, options)
+  }
+})
+</script>
+
+<style lang="scss" scoped>
+@import '@/styles/abstracts/variables.scss';
+
+.icon-box {
+  width: 2rem;
+  height: 2rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
`is-flex is-align-items-center is-justify-content-center`

Reply:
nvm just replace `icon-box` by `image is-24x24`

Reply:
>is-flex is-align-items-center is-justify-content-center

I normally go for this style 100%
in this case i felt that the class name would give an indication as to why it is there

>image is-24x24

i did, it doesn't center the calendar icon

Reply:
remove `mr-2` and add `has-text-centered`

Reply:
ok. thank you.
done